README.md

```diff
@@ -1 +1,84 @@
 # FightPandemics MongoDB Realm Apps
+
+This repo includes services built in [MongoDB Realm](https://www.mongodb.com/realm) to provide access to data stored in MongoDB Atlas for the [main FightPandemics app](https://github.com/FightPandemics/FightPandemics/).
+
+These apps are intended for use by FightPandemics team members and generally. They may be stop gap solutions until the desired functionality can be built into the main app itself.
+
+The front-end is currently Google Sheets. The code is defined within a `google-sheets/Code.js` file.
+
+## posts-moderation
+
+This app provides access to post content for FightPandemics team members to review the content for abuse. "Sourced by FightPandemics" posts are excluded.
+
+### Google Sheet Usage
+
+#### Granting Access
+
+The app is made available through a Google Sheet stored in the FightPandemics accounts Google Drive. Since G Suite is not being used users will be given access to this specific sheet by email invite.
+
+1. Get the email of the user requiring access for moderation.
+2. Open the "FightPandemics Moderation" sheet
+3. Click the "Share" button (top-right)
+4. Enter the email and add as an "Editor" and click "Send"
+
+#### Initial Usage & Granting Script Permissions
+
+Since G Suite is not being the script must be granted permission to run by each invited user.
+
+1. Click the link in invitation email
+2. Once the sheet fully loads click "Get Posts"
+3. Click "Continue" in prompt asking for permission to run
+4. A pop-up will open. Select your account & click "Advanced"
+5. Click "Allow"
```

Comment:
Is this supposed to be referencing the screenshot below? I don't see the "Allow" button in the screenshot.